content/riak/kv/2.1.4/using/cluster-operations/backing-up.md

```diff
@@ -20,50 +20,46 @@ aliases:
-inconsistent from node to node. Data could exist on some nodes and not
-others at the exact time a backup is made. Any inconsistency will be
-corrected once a backup is restored, either by Riak's [active anti-entropy](/riak/kv/2.1.4/learn/concepts/active-anti-entropy/) processes or when the object is read, via [read repair](/riak/kv/2.1.4/learn/concepts/active-anti-entropy/#read-repair-vs-active-anti-entropy).
-
-Additionally, backups must be performed on a stopped node to prevent
-data loss as a result of the background merging and compaction processes
-of Riak's backends. Downtime of a node can be significantly reduced by
-using an OS feature or filesystem that supports snapshotting.
+Riak KV is a [clustered][concept clusters] system built to survive a wide range of failure scenarios including the loss of nodes due to network or hardware failure. Although this is one of Riak KV's core strengths, it cannot withstand all failure scenarios.
+
+Backing up data; duplicating the database on a different long-term storage system, is a common approach to mitigating potential failure scenarios.
+
+The following document covers how to perform backups of Riak KV data.
+
+## Overview
+
+Riak KV backups can be performed using operating system features or filesystems that support snapshots, such as LVM or ZFS, or by using tools like rsync or tar.
+
+Choosing your Riak KV backup strategy will depend on your already-established backup methodologies and the backend configuration of your nodes.
+
+The basic process for getting a backup of Riak KV from a node is as follows:
+
+1. Stop the node with `riak stop`.
+2. Backup the appropriate data, ring, and configuration directories.
+3. Start the node.
+
+Downtime of a node can be significantly reduced by using an OS feature or filesystem that supports snapshotting.
+
+{{% note title="Backups and eventual consistency" %}}
+Due to Riak KV's eventually consistent nature, backups can become slightly inconsistent from node to node.
+
+Data could exist on some nodes and not others at the exact time a backup is made. Any inconsistency will be corrected once a backup is restored, either by Riak's [active anti-entropy](/riak/kv/2.1.4/learn/concepts/active-anti-entropy/) processes or when the object is read, via [read repair](/riak/kv/2.1.4/learn/concepts/active-anti-entropy/#read-repair-vs-active-anti-entropy).
+{{% /note %}}
 
 ## OS-Specific Directory Locations
 
-The default Riak data, ring, strong consistency, and configuration
-directories for each of the supported operating systems is as follows:
+The default Riak KV data, ring, and configuration directories for each of the supported operating systems is as follows:
 
-<div class="note">
-<div class="title">Note on upgrading</div>
-If you are upgrading to Riak version 2.0 or later from a pre-2.0
-release, you can use either your old <code>app.config</code>
-configuration file or the newer <code>riak.conf</code> if you wish.
+{{% note title="Note on upgrading" %}}
```

Comment:
So, this note only applies to 2.0 versions. It would not apply to 2.1.z or 2.2.0.